lib/util/object.ts

```diff
@@ -11,3 +13,20 @@ export function hasKey<K extends string, T>(
 ): o is T & Record<K, unknown> {
   return typeof o === 'object' && k in o;
 }
+
+export function filterUndefined(input: unknown): any {
```

Comment:
```suggestion
export function filterUndefined<T = any>(input: T): T {
```
So type is passed through and we don't need a typecast after calling this